### DISCUSSION TITLE
Elysia/Bun Support

### DISCUSSION THREAD

**Discussion by grantsingleton** [payloadcms/payload#3587] [Feature Requests & Ideas]
I'm working on Elysia support for Payload. I'm in research mode so nothing solid to talk about yet. First, I'm going to work on getting a next 13 app to run on Elysia. After that, hopefully we can build a plugin for Elysia to support Payload without having to change anything about payload. 

If anyone has made progress on any of this please contribute to the conversation! 

I'm especially interested in: 
* If you have ran next js on Elysia
* If you know of any blockers 
* Anything else related to the topic

**Comment by jmikrut** [comment]
This would be awesome. The Payload team is very interested in both Bun support and Elysia support!

**Comment by geminigeek** [comment]
its very interesting , i tried Elysia few week back , at that time i could not find any passportjs like implementation in Elysia , everything else is just in documents of  Elysia , i think there must be a solution like passportjs for Elysia , apart this i think, payload running on bun will be super

**Comment by revnelson** [comment]
Any progress on this? Where are you at? I'm interested in helping out but don't want to tackle things you've already figured out.

**Comment by jamesv-menklab** [comment]
Very interested in this as well.  I'd be willing to help out.
